.github/workflows/widgetbook_exception.yaml

```diff
@@ -0,0 +1,58 @@
+on: 
+  pull_request:
+    paths:
+      - "packages/widgetbook_exception/**"
+      - ".github/workflows/widgetbook_exception.yaml"
+  push:
+    branches:
+      - main
+name: widgetbook_exception
+jobs:
+  test:
+    name: test
+    defaults:
+      run:
+        working-directory: packages/widgetbook_exception
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        lfs: true
+    - name: Checkout LFS objects
+      run: git lfs checkout
+
+    - uses: subosito/flutter-action@v2
+      with:
+        channel: stable
+
```

Comment:
it's better to install dart action here because flutter takes a lot of time to install which will cause the action to run longer